demo/app.js

```diff
@@ -2,30 +2,34 @@ import Carousel from '../src/index';
 import React from 'react';
 import ReactDom from 'react-dom';
 
+const colors = ['7732bb', '047cc0', '00884b', 'e3bc13', 'db7c00', 'aa231f'];
+
 class App extends React.Component {
   constructor() {
     super(...arguments);
-    this.state = { slideIndex: 0 };
+    this.state = { slideIndex: 0, length: 6 };
   }
 
   render() {
     return (
       <div style={{ width: '50%', margin: 'auto' }}>
         <Carousel
           slideIndex={this.state.slideIndex}
-          afterSlide={newSlideIndex =>
-            this.setState({ slideIndex: newSlideIndex })
-          }
+          afterSlide={slideIndex => this.setState({ slideIndex })}
           renderTopCenterControls={({ currentSlide }) => (
-            <div>Nuka Carosel: Slide {currentSlide + 1}</div>
+            <div style={{ fontFamily: 'Helvetica', color: '#fff' }}>
+              Nuka Carousel: Slide {currentSlide + 1}
+            </div>
           )}
         >
-          <img src="http://placehold.it/1000x400&text=slide1" />
-          <img src="http://placehold.it/1000x400&text=slide2" />
-          <img src="http://placehold.it/1000x400&text=slide3" />
-          <img src="http://placehold.it/1000x400&text=slide4" />
-          <img src="http://placehold.it/1000x400&text=slide5" />
-          <img src="http://placehold.it/1000x400&text=slide6" />
+          {[...Array(this.state.length).keys()].map(key => (
```

Comment:
No IE support for this? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/keys

Side note -- let's add IE tests to func tests planning :)